Syntaxes/Julia.tmLanguage

```diff
@@ -244,13 +244,13 @@
 			<array>
 				<dict>
 					<key>match</key>
-					<string>\b(?:function|stagedfunction|type|immutable|macro|quote|abstract|bitstype|typealias|module|baremodule|new)\b</string>
```

Comment:
I should be the other way around. `new` is a keyword and the majority of the use are using it's keyword meaning so it's more important that those are highlighted correctly.